### PR TITLE
Fix advanced selection modals

### PR DESCRIPTION
**Pull request by gagnieray**

### CODE REVIEW

**Comment by gagnieray** [comment]
This PR doesn't fix [#1645](https://bugs.galette.eu/issues/1645) nor [#1646](https://bugs.galette.eu/issues/1646).

**Comment by gagnieray** [comment]
> This PR doesn't fix [#1645](https://bugs.galette.eu/issues/1645) nor [#1646](https://bugs.galette.eu/issues/1646).

725a8c2 and dc681b5 are attempts to handle those issues.

File: galette/templates/default/pages/transaction_form.html.twig [hunk 1]
```diff
@@ -170,17 +170,21 @@
         </p>
     </div>
 {% endif %}
+    <div id="contributions-modal" class="ui modal">
+        <div class="header"></div>
+        <div class="content"></div>
+        <div class="actions">
+            <button class="ui cancel button">{{ _T("Close")|e("js") }}</button>
```
Comment (trasher) [inline galette/templates/default/pages/transaction_form.html.twig:177]: I'm not sure for javascript escaping here

**Comment by trasher** [comment]
See https://github.com/galette/galette/pull/212

**Pull request closed without merging**